### PR TITLE
[B-16862] Phase III - Normalized Cluster Interface (NCI) - SPS-Linux Module

### PR DESCRIPTION
**Pull request by Gupta23**
Modify build_config_from_file(), change from protected to public to be accesed by NCIModule

### CODE REVIEW

File: python/SignaliQ/client.py [hunk 1]
```diff
@@ -39,7 +39,8 @@ def __init__(self, config = None):
         self._current_dir = None
         self._params = None
 
-        self._config = config if config else self._build_config_from_file()
+        config_file = path.join(self._get_current_dir(), "config.ini")
```
Comment (clemep-sios) [inline python/SignaliQ/client.py]: Can you change the name of _get_current_dir to _get_file_dirname (current dir is confusing, as that concept refers to the shell's current working directory, which is not necessarily the same as the directory path of this file)?

**Review by clemep-sios — CHANGES_REQUESTED** [review]
please make the requested change

File: python/SignaliQ/client.py (hunk 1 above)
Comment (Gupta23) [reply python/SignaliQ/client.py]: Changed the name to _get_file_dirname

**Comment by clemep-sios** [comment]
Looks good

**Review by srazin — APPROVED** [review]
looks good

**Review by clemep-sios — APPROVED** [review]

**Pull request merged**